hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1966,17 +1966,19 @@
       name: chip_sw_otbn_idle
       desc: '''Verify the OTBN idle signal to clkmgr.
 
+            - Write the OTBN clk hint to 0 within clkmgr to indicate OTBN clk and verify that the
+              OTBN clk hint status within clkmgr reads 0 (OTBN is idle).
             - Write the OTBN clk hint to 1 within clkmgr to indicate OTBN clk is ready to be gated.
```

Comment:
Just making sure I'm getting this right, but doesn't writing 1 mean that OTBN clock should not be gated? (I know this text existed already, but maybe it makes sense to get everything straight since we're touching it).